src/components/ThreeDNodeTree/ThreeDNodeTree.tsx

```diff
@@ -190,6 +194,13 @@ class ThreeDNodeTree extends React.Component<NodeTreeProps, NodeTreeState> {
     }
   };
 
+  onRightClickNode = (event: AntTreeNodeMouseEvent) => {
+    const { onRightClick } = this.props;
```

Comment:
Do you have some prop-validation? https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

Reply:
I have an interface for the props in NodeTypes.js, so I don't think any further validation is necessary.